RELEASE-NOTES.md

```diff
@@ -8,6 +8,7 @@
 
 ### New Features
 - The `CAR` distribution has been added to allow for use of conditional autoregressions which often are used in spatial and network models.
+- The shape handling of random variables was improved dramatically (see [#4625](https://github.com/pymc-devs/pymc3/pull/4625)).
```

Comment:
This isn't really a description of the changes; it's more like a statement about the quality of the (undescribed) changes.

Reply:
I should have used "changed" instead of "improved", yes. But as you can see above our entire v4 section in the release notes is practically nonexistent and will have to be worked anyway (out of scope).

For most changes we'll probably want to link a dedicated documentation page about v3/v4 differences, that's why I didn't bother to go into detail here.